proto/agent/v1/connection.proto

```diff
@@ -29,6 +30,19 @@ message UpdateAgentConnection {
     google.protobuf.Duration agent_uptime = 16;
 }
 
+message CloudCommand {
+    MessageType type = 1;
+    google.protobuf.Any payload = 2;
+}
```

Comment:
```suggestion
```

As we discussed, for now, there is no reason to wrap messages sent to the agent. The message type will be embedded in the message topic.